dobi.yaml

```diff
@@ -0,0 +1,93 @@
+
+meta:
```

Comment:
I think now that I got on a wrong track. I thought you said that dobi solved an installation abstraction issue because you mentioned overlap; what it does is simplify a docker install. Of course that gains us platform independence at some level. But the overlap you mentioned between dobi and fabric does not mean that one should replace the other, because the overlap is in very different layers. So we keep fabric for naked (dockerless) or within-container installs, and we use dobi for the docker install option (which is the only option on some platforms.) I'm happy with this scenario now that I am clear on it.